state/machine_upgradeseries_test.go

```diff
@@ -85,6 +85,15 @@ func (s *MachineSuite) TestRemoveUpgradeSeriesLockUnlocksMachine(c *gc.C) {
 	AssertMachineIsNOTLockedForPrepare(c, mach)
 }
 
+func (s *MachineSuite) TestRemoveUpgradeSeriesLockIsNoOpIfMachineIsNotLocked(c *gc.C) {
```

Comment:
This is needed since Removal of the lock may be prompted by an error which triggers cleanup. The error my be an error in the creation of the lock itself which would result in an attempt to remove a non-existent lock. An error which triggers cleanup can result in the attempted remove a non-existent lock, for this an other reasons we do not error when attempting to remove a non-existent lock.